op.h

```diff
@@ -162,6 +162,8 @@ Deprecated.  Use C<GIMME_V> instead.
                                 /*  On OP_UNSTACK, in a C-style for loop */
                                 /*  On OP_READLINE, it's for <<>>, not <> */
                                 /*  On OP_RETURN, module_true is in effect */
+                                /*  On OP_NEXT/OP_LAST/OP_REDO, there is no
+                                 *  loop label */
```

Comment:
Thanks! Though this also applies to goto and dump. dump is weird but mentioning it for goto is probs worth it